manager/integration/tests/test_node.py

```diff
@@ -113,7 +121,7 @@ def test_node_disk_update(client):  # NOQA
         # keep default disk for other tests
         if disk["path"] == disk_path1 or disk["path"] == disk_path2:
             disk["allowScheduling"] = False
-            disk["storageReserved"] = Gi
+            disk["storageReserved"] = 0
```

Comment:
Yeah, I think you should keep the original test if possible, like I said in the manager PR.

Reply:
This change is due to validation of disk storageReserved. My test disk size is 16MB, but when storageReserved is set to Gi, it couldn't pass this test.

Reply:
Hmm, why?

Reply:
Because we add a validation in node api, to validate `storageReserved` setting should no more than `storageMaximum`. If I keep the original test, it will return a validate exception, which has already added in disk update case.

Reply:
Yeah, I've commented on that earlier…